### PR TITLE
bump workspaces to v0.1.0-alpha8

### PR DESCRIPTION
**Pull request by filariow**
This PR bumps konflux-workspaces to `v0.1.0-alpha8` in staging

### CODE REVIEW

File: components/workspaces/staging/base/server/config/server/kustomization.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- deployment.yaml
+- service.yaml
```
Comment (sadlerap) [inline components/workspaces/staging/base/server/config/server/kustomization.yaml:5]: did `service.yaml` get deleted?

Comment (filariow) [reply components/workspaces/staging/base/server/config/server/kustomization.yaml:5]: no, it's there. It's easier to find it if you collapse the `proxy-config` folder.

File: components/workspaces/staging/base/server/config/server/service.yaml
Comment (filariow) [inline components/workspaces/staging/base/server/config/server/service.yaml:1]: @sadlerap

**Review by sadlerap — APPROVED** [review]
/lgtm

**Pull request merged**